.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+    greeting:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/first-interaction@v1
+                with:
+                    repo-token: ${{ secrets.GITHUB_TOKEN }}
+                    issue-message: '**Thanks for opening your first issue here!**<br/><br/>Be sure to follow the issue template! Your issue will be reviewed by a maintainer and labeled accordingly for further actions.'
```

Comment:
@m52go Could you go over this messages and suggest improvements if required? Thanks!